### ISSUE TITLE
Conflicting completion mechanisms

### ISSUE THREAD

**Issue by reagle** [Exafunction/codeium-sublime#3]
Perhaps this is touched on in #1 , but I notice that this doesn't work well with ST's native snippets and completions. For example, if I type very quickly, I'll get my native ST completion for `ril` (right image left CSS), if I take a moment and see Codium's suggestion, game over. I can never see the native completions again.

**Comment by clci** [comment]
I agree, this is a problem. I think that an option to set another key to accept Codium's suggestion instead of the tab key would be great.

**Comment by apoxwebmaster** [comment]
Any ideas, has anyone else had this happen? :-1:
